lib/plugins/features/movies/index.js

```diff
@@ -1,22 +1,39 @@
 'use strict';
 
-const Controller     = require('./controller');
-const MovieValidator = require('../../../validators/movie');
+const Controller        = require('./controller');
+const MovieValidator    = require('../../../validators/movie');
+const MovieGetValidator = require('../../../validators/movieget');
```

Comment:
I'd recommend created an intermediary directory for `movies` and then you can name the files `create` and `list`!